response/slack/decorators/view_handler.py

```diff
@@ -0,0 +1,79 @@
+import json
```

Comment:
This new handler allows us to register handlers for modals.  We make it easier for users to use them by parsing forms from submitted modals and providing a list of key, value pairs for the form.